converters/fromZigbee.js

```diff
@@ -857,6 +887,50 @@ const converters = {
             return payload;
         },
     },
+    command_ehanced_move_to_hue_and_saturation: {
+        cluster: 'lightingColorCtrl',
+        type: 'commandEnhancedMoveToHueAndSaturation',
+        convert: (model, msg, publish, options, meta) => {
+            const payload = {
+                action: getProperty(`enhanced_move_to_hue_and_saturation`, msg, model),
+                action_enhanced_hue: msg.data.enhancehue,
+                action_saturation: msg.data.saturation,
```

Comment:
We have enhanced hue with values from 0..65535 and standard hue from 0 to 359 which is reflected in the variable name. Saturation is a value here from 0 to 255 instead  a percentage value? Is this intended

Reply:
Will add `hue` and `saturation`.